Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/AirSinglePipe.cs

```diff
@@ -256,7 +294,19 @@ public override void Save(BinaryWriter outf)
             outf.Write(BleedOffValveOpen);
             outf.Write((int)HoldingValve);
             outf.Write(CylVolumeM3);
-        }
+            outf.Write(EmergResQuickRelease);
+            outf.Write(UniformChargingThresholdPSI);
+            outf.Write(UniformChargingRatio);
+            outf.Write(QuickServiceActive);
+            outf.Write(QuickServiceLimitPSI);
+            outf.Write(QuickServiceApplicationRatePSIpS);
+            outf.Write(QuickServiceVentRatePSIpS);
+            outf.Write(AcceleratedApplicationRatio);
+            outf.Write(AcceleratedApplicationLimitPSIpS);
+            outf.Write(InitialApplicationThresholdPSI);
+            outf.Write(BrakeCylinderSpringPressurePSI);
+            outf.Write(ServiceMaxCylPressurePSI);
```

Comment:
You don't need to save things that are constant (e.g. application rates, volumes, etc.). Only things that may change during the simulation need to be changed (e.g. QuickServiceActive or current cyl pressure).

Reply:
Good to know, I wasn't entirely sure how saving interacted with initialization-I'll get rid of those things that don't need to be re initialized.